resources/views/content/contributor.blade.php

```diff
@@ -13,10 +13,51 @@
             <imgix :src="$avatar_url" width="256" height="256" ratio="1:1" :alt="$login.' Avatar'" class="mx-auto rounded w-64 h-64" />
             <h2 class="text-6xl text-white font-bold">{{ $login }}</h2>
 
+            @if(isset($info['bio']))
+            <div class="py-4">  
+                <p class="mt-4">{{$info['bio']}}</p>
+            </div>
+            @endif
+
             <div class="flex flex-row flex-wrap justify-center">
                 <count-up icon="fa-box-heart" :value="count($packages)" label="packages" />
                 <count-up icon="fa-code-commit" :value="$commits" label="commits" />
             </div>
+
+            <div class="mx-auto max-w-3xl">
```

Comment:
I think if we add these details we are good to add the GitHub profile link as well!? Possibly use the `info.name` in the headline and fallback to the username if it's undefined.
As the real names are longer we should reduce the headline size from `6xl` to 5 or 4 - I think Freek is a good profile to test this as he has a long name.